crates/cli/Cargo.toml

```diff
@@ -1,17 +0,0 @@
-[package]
```

Comment:
so cli was moved to [crates/starknet-server/src/main.rs](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/26/files#diff-b91426894da81bbc09dd3a1debdf35a66c05e724fc2c75a7ecc33651192ecca5)?

Reply:
ok thanks, got it

Reply:
are you talking to yourself or what?

Reply:
because we don't need it anymore, either Starknet-server should be moved to cli or cli to Starknet-server. I choose cli -> Starknet-server, because its less code to move